### PR TITLE
fix: adjust landing page for rmrk2

### LINKED ISSUE #5196: Showing RMRK2 latest lists on landing
Show latest sales in latest sales strip from RMRK2

image

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5196
- [x] hide top collection on rmrk2

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

Screenshot 2023-03-24 163122

### CODE REVIEW

File: components/carousel/utils/useCarousel.ts [hunk 1]
```diff
@@ -28,11 +28,11 @@ interface Types {
...
 const nftEventVariables = {
   latestSales: {
-    limit: 30,
+    limit: 8,
     event: 'BUY',
   },
   newestList: {
-    limit: 30,
+    limit: 8,
     event: 'LIST',
```
Comment (vikiival) [inline components/carousel/utils/useCarousel.ts:36]: ?

**Comment by vikiival** [comment]
It's still indexing ^-^ 

Screenshot 2023-03-24 at 14 06 11

**Comment by exezbcz** [comment]
image
- missing the network indicator in the bottom right corner - rmrk2

**Comment by vikiival** [comment]
> image
> 
> - missing the network indicator in the bottom right corner - rmrk2

Its kusama

**Review by vikiival — APPROVED** [review]

**Comment by exezbcz** [comment]
@vikiival and how will kusama be named (rmrk1)

**Review by roiLeo — COMMENTED** [review]
It show only BSX & Snek Rococo items on LatestList & LastestSales for me

is it excepted behavior? maybe indexer issue

**Comment by yangwao** [comment]
> what if we fetch only 8 items for each chain? so we can see all the latest nft from each chain in that carousel?

works I guess for now

I've made now list of this NFT so should be visible, but probably indexer is still stuck? @vikiival 

https://beta.kodadot.xyz/rmrk2/gallery/15174172-dac5c7f54029d0e73c-6AFVH-DAWN-00001250

**Comment by yangwao** [comment]
> what if we fetch only 8 items for each chain? so we can see all the latest nft from each chain in that carousel?

indexer should be close to indexed state so maybe we can start seeing something?

```
"syncStatus": {
"totalBlocks": 17238206,
"currentBlock": 16376765
```

**Comment by yangwao** [comment]
> is that ok?

yes

> it's not a good idea to merge this one if we still have problems fetching rmrk2 images

rmrk2 supposed to be on beta only

**Comment by yangwao** [comment]
let's fix conflicts and I mean it's good to go, we will look and speed up loading of sales 
- #5422

**Review by yangwao — APPROVED** [review]

**Comment by vikiival** [comment]
Screenshot 2023-03-29 at 17 18 02

**Comment by yangwao** [comment]
pay 30 usd 

let's say it's there 

image

**Pull request merged**